manifests/webhook/config.pp

```diff
@@ -20,6 +20,7 @@
   $use_mco_ruby                   = $r10k::params::webhook_use_mco_ruby,
   $protected                      = $r10k::params::webhook_protected,
   $github_secret                  = $r10k::params::webhook_github_secret,
+  $bitbucket_secret               = $r10k::params::webhook_bitbucket_secret,
```

Comment:
Can you please add a proper datatype for the new parmeter?

Reply:
Thanks for your review! Fixed it.